### ISSUE TITLE
[Feature request] ignore list for custom-classes

### ISSUE THREAD

**Issue by thebuilder** [francoismassart/eslint-plugin-tailwindcss#16]
The `custom-classes` is a nice way of finding invalid classes, but it would be nice with the option of supporting some custom classes. Right now it's all or nothing.	

As an example, I've added a `.focus-outline` to our CSS that gets flagged as custom:

```css
@layer components {
  .focus-outline {
    @apply focus:outline-none focus-visible:ring-2 focus-visible:ring-aqua;
  }
}
```

**Comment by francoismassart** [comment]
Great idea, this should not be complicated to implement.

**Comment by francoismassart** [comment]
There you go @thebuilder, `v1.6.0` freshly released on both [github](https://github.com/francoismassart/eslint-plugin-tailwindcss/releases/tag/v1.6.0) and [npm](https://www.npmjs.com/package/eslint-plugin-tailwindcss) with the optional `whitelist` option available.

Thank you for sharing your feature request 👍

**Comment by thebuilder** [comment]
Amazing! Thanks for that, and great plugin so far. Seems to be working great🙌
